crates/cdk/src/publish.rs

```diff
@@ -142,9 +147,21 @@ pub fn package_push(opts: &PublishCmd, pkgpath: &str, access: &HubAccess) -> Res
     Ok(())
 }
 
-pub fn init_package_template(package_info: &PackageInfo, binary_arch: &str) -> Result<()> {
+pub fn init_package_template(
+    package_info: &PackageInfo,
+    binary_arch: &str,
+    readme_path: &Option<PathBuf>,
+) -> Result<()> {
     let connector_toml_path = find_connector_toml(package_info)?;
     let connector_metadata = ConnectorMetadata::from_toml_file(&connector_toml_path)?;
+    let readme_md_path = if let Some(readme_path) = readme_path {
```

Comment:
This would be easier if option is mandatory

Reply:
This is the commit in question: https://github.com/infinyon/fluvio/pull/3230/commits/b735e845d42d2700bdfdbab105c4c922a3ba1881